packages/material-ui/src/ButtonBase/Ripple.js

```diff
@@ -20,7 +21,7 @@ function Ripple(props) {
   } = props;
   const [leaving, setLeaving] = React.useState(false);
 
-  const rippleClassName = clsx(classes.ripple, classes.rippleVisible, {
+  const rippleClassName = clsx(className, classes.ripple, classes.rippleVisible, {
```

Comment:
We need to apply `className` from props to, in order for the styles coming from `emotion` to be applied.